Composer/packages/server/src/constants.ts

```diff
@@ -18,3 +18,5 @@ export enum ClaimNames {
 export const APPINSIGHTS_INSTRUMENTATIONKEY = process.env.APPINSIGHTS_INSTRUMENTATIONKEY;
 
 export const piiProperties = ['sessionId', 'userId', 'projectId'];
+
+export const COMPOSER_VERSION = '1.2.0';
```

Comment:
let's take a look as a best effort to unify the version in one place if possible.

Reply:
I can create a new PR to fix this